src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializer.java

```diff
@@ -199,8 +202,16 @@ private String validateSortClausePath(String path) {
         throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
             "path must be represented as a non-empty string");
       }
-      throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
-          "path ('%s') cannot start with `$`", path);
+      // But allow "well-known" fields
+      switch (path) {
```

Comment:
Since this method is called for all entries, needs to let legal "$ fields" to pass.